Demo/Components/Viewings/ViewingsDemoView.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import FinniversKit
+
+class ViewingsDemoView: UIView {
+
+    private lazy var viewingsView = ViewingsView(frame: .zero)
```

Comment:
you can just say

`private lazy var viewingsView = ViewingsView(withAutoLayout: true)`

and remove the `viewingsView.translatesAutoresizingMaskIntoConstraints`